appveyor/scripts/tests/beforeTests.ps1

```diff
@@ -2,4 +2,14 @@ New-Item -ItemType directory -Path testOutput
 New-Item -ItemType directory -Path testOutput\unit
 New-Item -ItemType directory -Path testOutput\system
 New-Item -ItemType directory -Path testOutput\lint
+# The first system test to run that requires chrome occasionally fails.
+# This has been replicated on developer machines after chrome updates,
+# however it is difficult to reproduce, and difficult to detect/recover-from
+# in an automated way.
+# It may be possible to handle this better within NVDA, however it seems to actually affect users
+# infrequently, and they seem to be able to recover by waiting a few seconds and de/refocus chrome.
+# In the mean time, the system tests failing delays development.
+#
+# Theory: Chrome is busy with post install tasks, so start chrome in the background ahead of the tests.
+cmd /c start /min chrome.exe

```

Comment:
In `ChromeLib.py`, Chrome is opened with the following parameters:
* `--force-renderer-accessibility`
*  `--suppress-message-center-popups`
* `--disable-notifications`
* `--no-experiments`
* `--no-default-browser-check`
* `--lang=en-US`

Regarding `--lang=en-US`, I remember that this parameter only has effect when Chrome is first opened, not if there are other Chrome windows already opened. This parameter is not important on appVeyor since Chrome is already in English on appVeyor.

However, if the other parameters behave the same way, the risk is that they will not be taken into account during future tests since Chromme will already be pre-opened with no parameters before.

Please double-check and pass the same parameters as in `ChromeLib.py` if needed.

Reply:
Yep, this seems like a good idea.